chart/compass/charts/gateway/templates/oathkeeper-rules.yaml

```diff
@@ -438,7 +438,7 @@ spec:
   upstream:
     url: "http://compass-director.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.global.director.graphql.external.port }}"
   match:
-    methods: ["PATCH"]
+    methods: ["PATCH", "POST", "DELETE"]
```

Comment:
PATCH seem to be unused now, should we remove it?

Reply:
It is used for the formation assignment status API. There we use PATCH requests. The new ones - POST and DELETE are needed for the "new" formation status API